<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-29T14:21:45 IST" name="Default suite" finished-at="2023-05-29T17:45:32 IST" duration-ms="12227130">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:Runner.TestApplication@53045c6c]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-05-29T14:21:45 IST" name="Default test" finished-at="2023-05-29T17:45:32 IST" duration-ms="12227130">
      <class name="Runner.TestApplication">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:Runner.TestApplication@53045c6c]" started-at="2023-05-29T14:21:45 IST" name="setUpClass" finished-at="2023-05-29T14:21:50 IST" duration-ms="4694" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:Runner.TestApplication@53045c6c]" started-at="2023-05-29T14:21:50 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-29T17:44:57 IST" duration-ms="12187288" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Checking Cash Deposit Transaction - FC/ CV"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Checking the functionalities of transaction]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=113.0.5672.127)
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [dd049705ede8d98cb3d245c43b1a5210, findElement {using=xpath, value=//td[text()='USD']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.127, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\ININDC~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57987}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:57987/devtoo..., se:cdpVersion: 113.0.5672.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: dd049705ede8d98cb3d245c43b1a5210]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=113.0.5672.127)
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [dd049705ede8d98cb3d245c43b1a5210, findElement {using=xpath, value=//td[text()='USD']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.127, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\ININDC~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57987}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:57987/devtoo..., se:cdpVersion: 113.0.5672.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: dd049705ede8d98cb3d245c43b1a5210
at org.testng.Assert.fail(Assert.java:99)
at stepdefinitions.CSM_TransactionsSteps.user_enter_the_amount_in_maintenance_screen(CSM_TransactionsSteps.java:225)
at âœ½.User Enter the Amount in Maintenance Screen(file:///C:/Users/inindc00612/git/AZENTIO-CSM/CSM_RAZIA2/src/test/java/features/CSM_transactions.feature:73)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:Runner.TestApplication@53045c6c]" started-at="2023-05-29T17:44:58 IST" name="tearDownClass" finished-at="2023-05-29T17:45:32 IST" duration-ms="34834" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- Runner.TestApplication -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
